cluster/gce/config-default.sh

```diff
@@ -83,7 +83,7 @@ fi
 # you are updating the os image versions, update this variable.
 # Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
```

Comment:
wow... the versions at image-config.yaml are really old...

Reply:
:see_no_evil: Can I please unsee that? ... No, fine then please file a bug and send it my way... :speak_no_evil: